lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/reuse-existing-identity.yaml

```diff
@@ -9,6 +9,9 @@ START:
           targetState: CRI_TICF_BEFORE_REUSE
         deleteDetailsEnabled:
           targetState: IDENTITY_REUSE_PAGE_TEST
+        sixMonthFraudCheckEnabled:
+          targetJourney: SIX_MONTH_FRAUD_CHECK
+          targetState: START
```

Comment:
👍  - just need to make sure this is removed as part of Sammy's ticket!

Reply:
@sammyiyke Are you fine to remove this as part of your ticket? 

(this is just a testing journey so we can access/test the new pages while you are working on the real trigger for this flow)